.github/ISSUE_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ If it is a bug report:
 You can install the latest version from https://github.com/PowerShell/PowerShell/releases
 - Search the existing issues.
 - Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
-- Refer to the [known issues](https://github.com/PowerShell/PowerShell/blob/master/docs/KNOWNISSUES.md).
+- Refer to the [known issues](https://docs.microsoft.com/en-us/powershell/scripting/whats-new/known-issues-ps6?view=powershell-6).
```

Comment:
Maybe we should drop the `en-us` part from the url. So it will redirect to appropriate language according to locale. @sdwheeler am I correct?